src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/Microsoft.Azure.Management.RecoveryServices.Backup.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.RecoveryServices.Backup</PackageId>
 		<Description>Provides developers with libraries for the updated recovery services backup feature under Azure Resource manager.</Description>
-		<Version>3.0.0-preview</Version>
+		<Version>3.0.1-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.RecoveryServices.Backup</AssemblyName>
 		<PackageTags>Recoveryservices;Backup;</PackageTags>
 		<PackageReleaseNotes/>
```

Comment:
Please add meaningful information in the PackageReleaseNotes similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj#L13)